docs/example_python.md

```diff
@@ -1,30 +1,194 @@
 ---
-title: "Example: Python"
+title: "Example: Python + Flask"
 layout: docs
 ---
 
-You're a Python developer. You want to set up Tilt with your project.
+The best indicator of a healthy development workflow is a short feedback loop.
 
-We hear you! We're working on a better guide for this.
+Kubernetes is a huge wrench in the works.
 
-For now, check out:
+Let's fix this.
 
-- [abc123](https://github.com/windmilleng/abc123) a mini microservice app with a Python server called `numbers`.
-- Servantes (below), our microservice app in multiple languages.
+In this example, we're going to take you through a simple “hello world” server written in Python that uses [Flask](https://palletsprojects.com/p/flask/), a lightweight web application framework.  (Want this guide for Django or another framework? [Let us know](https://tilt.dev/contact).)
 
-## Servantes
+We'll use Tilt to:
 
-[Servantes](https://github.com/windmilleng/servantes) is a personalized homepage
-app implemented with many microservices in multiple languages. You can run it
-with the commands
+- Run the server on Kubernetes
+- Measure the time from a code change to a new process
+- Optimize that time for fast feedback
 
+Obviously, this is a silly example. But it can be a useful example to confirm that Tilt is working
```

Comment:
Let's axe this line, it was specific to busybox. Flask isn't silly at all!

Reply:
I mean, it's a silly hello world server -- but, fair